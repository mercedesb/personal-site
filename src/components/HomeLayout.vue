<template>
  <router-view v-cloak :key="$route.fullPath" />
</template>

<script>
export default {}
</script>

<style lang="scss">
</style>
