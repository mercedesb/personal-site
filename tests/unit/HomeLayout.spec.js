import HomeLayout from '@/components/HomeLayout.vue'
import { TestUtility } from './TestUtility'

describe('HomeLayout', () => {
  let component

  const initialProps = {
    color: 'brown',
    title: 'title',
    preamble: 'heres the preamble',
    media: 'http://image.com'
  }

  const shallow = propsData => TestUtility.shallow(HomeLayout, {
    propsData: {
      ...initialProps,
      ...propsData
    }
  })

  describe('Snapshots', () => {
    it('matches snapshot', () => {
      component = shallow()
      expect(component.element).toMatchSnapshot()
    })
  })
})
