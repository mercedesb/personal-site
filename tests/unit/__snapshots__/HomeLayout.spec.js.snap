// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLayout Snapshots matches snapshot 1`] = `
<router-view-stub
  color="brown"
  media="http://image.com"
  preamble="heres the preamble"
  title="title"
/>
`;
